src/effects/VST3/VST3Effect.cpp

```diff
@@ -0,0 +1,1138 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file VST3Effect.cpp
+
+  @author Vitaly Sverchinsky
+
+  @brief Part of Audacity VST3 module
+
+**********************************************************************/
+
+#include "VST3Effect.h"
+
+#include "Base64.h"
+
+#include "widgets/AudacityMessageBox.h"
```

Comment:
Avoid new calls to `AudacityMessageBox`.  Instead include `BasicUI.h` and use `BasicUI::MessageBox`.  Leave fewer calls to rewrite later when we try to make this toolkit neutral.